dGame/dComponents/BaseCombatAIComponent.h

```diff
@@ -388,5 +387,3 @@ class BaseCombatAIComponent : public Component {
 	 */
 	bool IsMech();
 };
-
-#endif // BASECOMBATAICOMPONENT_H
```

Comment:
as an fyi, because we use PCH, removing traditional include guards may cause multiple definition rule (ODR) violations.  Be wary of removing traditional guards.

Reply:
I'll revert those changes in the next commit